### DISCUSSION TITLE
Adaptive text using the Typography from Ant design 5

### DISCUSSION THREAD

**Discussion by evgeniyworkbel** [ant-design/ant-design#41328] [Q&A]
How do I make adaptive text using the Typography component from Ant design 5?

The Typography component has 3 elements: Title, Text Paragraph. How can I make the text they render shrink/enlarge according to the current screen resolution?

Now text rendered by Typography is not responsive.

**Comment by cliancarlos** [comment]
Hi, have you tried using the `vw` unit from the css? [(more info)](https://developer.mozilla.org/en-US/docs/Web/CSS/Viewport_concepts)

I believe that combined with `@media` for you to have more control it can work.

Example:
`<Title level={2} style={{ fontSize: '3vw' }}>Title of example</Title>`

**Reply by evgeniyworkbel** [reply]
@cliancarlos it is a good choice to adapt a few lines of text like titles, but to order to make adaptive websight you need to set all sizes in relative measuring units like em, rem
